template-parts/content-page.php

```diff
@@ -15,9 +15,7 @@
 		</div>
 	<?php endif; ?>
 
-	<header class="entry-header">
-		<?php the_title( '<h1 class="entry-title">', '</h1>' ); ?>
-	</header>
+	<?php alcatraz_entry_title(); ?>
```

Comment:
Even though our hide_page_title option only applies to pages, I think we should switch to calling this function across all of our content-*.php templates, especially if we add the `alcatraz_entry_title` filter. This will allow us to make global mods there to affect all post types, and of course we can always check the post type and only do something on pages or a cpt also.